source/_docs/configuration/templating.markdown

```diff
@@ -107,6 +107,18 @@ If your template uses an `entity_id` that begins with a number (example: `states
 Rendering templates with time (`now()`) is dangerous as updates only trigger templates in sensors based on entity state changes.
 </p>
 
+## {% linkable_title Priority of operators %}
+
+The default priority of operators is that the filter (`|`) has priority over everything except brackets. This means that:
```

Comment:
I would make it `The default priority of operators is the filter (`|`) has priority over everything except brackets. This means that:` The `that` just didn't flow well